cpp/ql/lib/semmle/code/cpp/exprs/Expr.qll

```diff
@@ -58,6 +58,21 @@ class Expr extends StmtParent, @expr {
   /** Gets the parent of this expression, if any. */
   Element getParent() { exprparents(underlyingElement(this), _, unresolveElement(result)) }
 
+  /**
+   * Gets the `n`th compiler-generated destructor call that is performed after this expression, in
+   * order of destruction.
+   */
```

Comment:
I think it would be good to have an example here (specifically, for when there are multiple destructors attached to an element).

Reply:
I think this would still be good, but given that we don't actually generate any such destructor calls yet this can wait 🙈